apps/shipments/views.py

```diff
@@ -231,7 +229,7 @@ def create(self, request, *args, **kwargs):
 
             send_templated_email('email/shipment_link.html', subject, email_context, emails)
 
-        return Response(serializer.data, status=status.HTTP_201_CREATED)
+        return Response(PermissionLinkSerializer(permission_link).data, status=status.HTTP_201_CREATED)
```

Comment:
I think I'd prefer to keep the usage of `self.get_serializer` instead of referencing the serializer directly

Reply:
@jamesfneyer

Reply:
Whoops! Thought I pushed this change last week :grimacing:

Reply:
@ajhodges check it now